fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -3629,6 +3637,7 @@ public void testLoanScheduleWithInterestRecalculation_WITH_REST_WEEKLY_INTEREST_
 
     }
 
+    @Disabled // FINERACT-1050
```

Comment:
Fix these before we merge? There are too many to just merge this without the tests.

Reply:
Ok. I have no problem handling this now. Just didn't want the PR to have too many different changes for easy review. Will work on it this week.